utils/__init__.py

```diff
@@ -71,84 +72,165 @@ def get_entry_format(file_path, file_type):
 
     return file_entry_format
 
-def create_omex(sedml_file, sbml_file):
+
+def add_xmlns_sbml_attribute_if_missing(sedml_filepath, sbml_filepath):
```

Comment:
Is this method a duplication of the method below?

Reply:
Yes, Rob split up the "add_xmlns_sbml_attribute_if_missing" I created into "add_xmlns_sbml_attribute" and "xmlns_sbml_attribute_missing". I think we can delete "add_xmlns_sbml_attribute_if_missing".